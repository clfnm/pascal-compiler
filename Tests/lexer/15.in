a+b
?&*@!
"b7
и