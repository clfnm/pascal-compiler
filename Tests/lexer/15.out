1	1	Id	a	a
1	2	Operation	Addition	+
1	3	Id	b	b
1	4	Separator	Dot	.
1	5	Eof	EOF	